### PR TITLE
Add XrostSSP support for amp-ad

### PR DESCRIPTION
**Pull request by optssp**
Please review this pull request for adding XrostSSP support on amp-ads.

### CODE REVIEW

**Comment by optssp** [comment]
I signed it!

**Comment by lannka** [comment]
@optssp can you take a look at [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request) and implement render-start , no-content API?

File: ads/xrostssp.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# Xrost SSP
+
+## Example
+
+```html
+<amp-ad width="320" height="50"
+    type="xrostssp"
+    data-aid="10"
+    data-slot_id="3347">
```
Comment (lannka) [inline ads/xrostssp.md]: you can do `data-slot-id`, so you will get it as `slotId` in the script

File: ads/xrostssp.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function xrostssp(global, data) {
+
+  validateData(data, [], ['aid', 'slot_id']);
+  
+  const url = 'https://proparm.jp/ssp/p/js1'
+  + '?_aid=' + encodeURIComponent(data.aid)
```
Comment (lannka) [inline ads/xrostssp.js]: please indent 2 more spaces

File: examples/ads.amp.html [hunk 3]
```diff
@@ -1101,6 +1102,15 @@ <h2 id="xlift">Xlift native ad</h2>
       <div placeholder></div>
       <div fallback></div>
   </amp-ad>
+  
+  <h2 id="xrostssp">Xrost SSP</h2>
+  <amp-ad width=300 height=250
+      type="xrostssp"
+      data-aid="10"
+      data-slot_id="3347"
+  <div placeholder></div>
```
Comment (lannka) [inline examples/ads.amp.html]: please fix indentation

**Comment by optssp** [comment]
@lannka Thank you for your reply!
Is it required to implement this API?

**Comment by lannka** [comment]
@optssp not mandatory, but highly recommended. As it provides much better user experience.

File: ads/xrostssp.md (hunk 1 above)
Comment (optssp) [inline ads/xrostssp.md]: I implemented to your advise.

**Comment by optssp** [comment]
@lannka Could you tell me why travis failed?

**Comment by optssp** [comment]
I rebased.
Please check it!

**Comment by optssp** [comment]
@lannka I fixed travis failed.How we can fix CLA issue now?

**Comment by optssp** [comment]
The CLA has been signed as OPT, Inc.

File: ads/_config.js [hunk 4]
```diff
@@ -466,6 +466,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  xrostssp: {},
```
Comment (lannka) [inline ads/_config.js]: do you want to leverage  preconnect https://proparm.jp?

File: ads/xrostssp.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function xrostssp(global, data) {
+
+  validateData(data, [], ['aid', 'slotId']);
+
+  const url = 'https://proparm.jp/ssp/p/js1'
+    + '?_aid=' + encodeURIComponent(data['aid'])
```
Comment (lannka) [inline ads/xrostssp.js]: please indent 2 more spaces

**Comment by optssp** [comment]
@lannka I signed CLA as a company and the company's name is OPT, Inc.
Could you please check if there's our company's name on the CLA list? If not, could you tell us what to do in order to pass googlebot's verification?

File: ads/_config.js (hunk 4 above)
Comment (optssp) [inline ads/_config.js]: I don't need it because I don't use JS.

**Comment by lannka** [comment]
OPT Inc is on the list. The robot sucks.

@erwinmombay , could you help force merge?

**Pull request merged**